### PR TITLE
Internal: Added lighthouse-playwright plugin [ED-12181]

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/playwright/sanity/lighthouse.test.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { playAudit } from 'playwright-lighthouse';
+import { test } from '@playwright/test';
+import config from 'lighthouse/lighthouse-core/config/desktop-config';
+
+test.describe( 'audit example', () => {
+	test( 'open browser', async ( { page } ) => {
+		await page.goto( 'http://localhost:8888/law-firm-about/' );
```
Comment (hein-obox) [inline tests/playwright/sanity/lighthouse.test.ts:7]: One question about this: how can we test this for widgets?

Could we create a new page, add our content, publish the page, and test the page?
And remove the page afterwards?

I will approve this PR, once I know how we can use it.

Comment (hein-obox) [reply tests/playwright/sanity/lighthouse.test.ts:7]: @dmaziar

Comment (dmaziar) [reply tests/playwright/sanity/lighthouse.test.ts:7]: yes, this sounds good. I'll extend this test with widget.

Comment (dmaziar) [reply tests/playwright/sanity/lighthouse.test.ts]: Updated. Added create a page, load template from element regression, publish and then do lighthouse check. 
Ideas, comments are welcome!

File: tests/playwright/sanity/lighthouse.test.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		await playAudit( {
+			page,
+			config,
+			thresholds: {
+				performance: 85,
+				accessibility: 85,
+				'best-practices': 85,
+				seo: 80,
+			},
+			port: 9222,
```
Comment (yotamselementor) [inline tests/playwright/sanity/lighthouse.test.ts:24]: Can we make the `port` a constant that we use both here and in the playwright config?

File: tests/playwright/sanity/lighthouse.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { playAudit } from 'playwright-lighthouse';
+import { test } from '@playwright/test';
+import config from 'lighthouse/lighthouse-core/config/desktop-config';
+import WpAdminPage from '../pages/wp-admin-page';
+import _path from 'path';
+
+test.describe( 'Lighthouse tests', () => {
+	test( 'Accordion widget test', async ( { page }, testInfo ) => {
```
Comment (yotamselementor) [inline tests/playwright/sanity/lighthouse.test.ts:8]: If we add more tests here, many/most/all of them will be in this format, we should think about extracting this to a function in the future (not for this PR)

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/lighthouse.test.ts (hunk 2 above)
Comment (dmaziar) [reply tests/playwright/sanity/lighthouse.test.ts]: fixed

File: tests/playwright/sanity/lighthouse.test.ts (hunk 3 above)
Comment (dmaziar) [reply tests/playwright/sanity/lighthouse.test.ts:8]: yes

**Review by yotamselementor — APPROVED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**